website/server/controllers/api-v4/admin.js

```diff
@@ -1,8 +1,11 @@
 import validator from 'validator';
+import _ from 'lodash';
```

Comment:
The only lodash function we use here is `merge` -- can we import merge directly instead of the whole library?